src/coldcard.js

```diff
@@ -211,26 +220,17 @@ export class ColdcardExportPublicKey extends ColdcardFileParser {
   }
 
   messages() {
-    const messages = super.messages();
-    messages.unshift({
-      state: PENDING,
-      level: INFO,
-      code: "coldcard.export",
-      text: "Go to Settings > Multisig Wallets > Export XPUB",
-    });
-    return messages;
+    return super.messages();
   }
 
   parse(xpubJSONFile) {
     const result = super.parse(xpubJSONFile);
-    let xpub = super.deriveXpubIfNecessary(result.xpub);
-
+    let xpub = super.deriveXpubIfNecessary(result.p2sh);
```

Comment:
Is the `super` necessary here? This class is already inheriting from the parent and so it should already have `this.deriveXpubIfNecessary` unless the parent's version of it is different then what's on this child class? (super is necessary for `parse` since we're in this class' own custom `parse`)

Reply:
Good catch.